crest/Assets/Crest/Crest/Scripts/Underwater/UnderwaterRenderer.Mask.cs

```diff
@@ -64,6 +66,21 @@ void OnPreRenderOceanMask()
             );
         }
 
+        internal static void AdjustNearPlaneForMSAA(Camera camera, CommandBuffer buffer)
+        {
+            if (Helpers.IsMSAAEnabled(camera))
+            {
+                // @MSAAOutlineFix:
+                // The edge of the ocean surface at the near plane will be MSAA'd leaving a noticeable edge. By rendering
```

Comment:
Interesting, thanks for the note. If this offset is larger, does it cause any issue or just push it back gracefully? Just checking if the offset should be proportional to frustum length or such or if approximate is fine.

Reply:
In certain cases where the near plane intersects a wave, like when looking down, the darkened outline is much thicker so making it larger would make that case worse.

What is the benefit in making it proportional?

Reply:
Ah ok. Was just wondering if it would help us to work on a wider range of scenarios. However what you describe wouldn't be helped by my suggestion and I think it's fine as is.